pkg/operator/controller.go

```diff
@@ -574,13 +625,12 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 
 	configInformerFactory.Start(stopCh)  // ClusterOperator
 	kubeNTOInformerFactory.Start(stopCh) // DaemonSet
-	kubeInformerFactory.Start(stopCh)    // Pod/Node
+	kubeInformerFactory.Start(stopCh)    // Node
 	tunedInformerFactory.Start(stopCh)   // Tuned/Profile
 
 	// Wait for the caches to be synced before starting worker(s)
 	klog.Info("waiting for informer caches to sync")
 	ok := cache.WaitForCacheSync(stopCh,
-		podInformer.Informer().HasSynced,
```

Comment:
Do we still wait for the podInformer cache sync if the pod labels are enabled?

Reply:
We do not at the moment.  We can, however, the question is whether we want to.  On a small cluster I've measured the cache sync wait on enabling the Pod sync to be 0.5-1s.  I cannot think of a good reason to enable it, but there might be one.  Can you think of anything?

Reply:
Just wanted to make sure it was still functional as is.